src/main/resources/hibernate.cfg.xml

```diff
@@ -4,18 +4,19 @@
         "http://www.hibernate.org/dtd/hibernate-configuration-3.0.dtd">
 <hibernate-configuration>
     <session-factory>
+
         <!-- Database connection settings -->
-        <property name="connection.driver_class">org.postgresql.Driver</property>
-        <property name="dialect">org.hibernate.dialect.CockroachDB201Dialect</property>
-        <property name="connection.url">jdbc:postgresql://127.0.0.1:26257/bank?sslmode=disable</property>
-        <property name="connection.username">maxroach</property>
-        <property name="connection.password"></property>
+        <property name="hibernate.connection.driver_class">org.postgresql.Driver</property>
+        <property name="hibernate.dialect">org.hibernate.dialect.CockroachDB201Dialect</property>
+        <property name="hibernate.connection.url">jdbc:postgresql://localhost:26257/bank?ssl=true&amp;sslmode=require</property>
+        <property name="hibernate.connection.username">maxuser</property>
```

Comment:
Just a nit:
Should we just make this username `user`? I don't think we use `maxuser` as a username anywhere in the docs.

Reply:
I hadn't pushed the commit with that fix.